### ISSUE TITLE
[Feature Request]: add linux/arm64 docker build

### ISSUE THREAD

**Issue by rsubr** [grokability/snipe-it#12795]
### Is your feature request related to a problem? Please describe.

Thank you for the great project :-)

Docker Hub images are currently for linux/amd64 only. Please add linux/arm64 builds so we can run SnipeIT on more cloud server platforms.

As GH Actions is using docker buildx, adding arm64 builds should be a trivial fix.

### Describe the solution you'd like

GH Actions files:
 - https://github.com/snipe/snipe-it/blob/develop/.github/workflows/docker.yml
 - https://github.com/snipe/snipe-it/blob/develop/.github/workflows/docker-alpine.yml

Pls add `linux/arm64` to the list of buildx platforms.

```yaml
      # https://github.com/docker/build-push-action
      - name: Build and push 'snipe-it' image
        id: docker_build
        uses: docker/build-push-action@v4
        with:
          ...
          platforms: linux/amd64, linux/arm64
          ...
```

Thanks!

**Comment by snipe** [comment]
@uberbrady I’m assigning this one to you - if you feel we have a team member (who is not currently on vacation) that’s better suited, please redirect

**Comment by snipe** [comment]
Fixed in #12904
